### ISSUE TITLE
jabref.xml not saved 

### ISSUE THREAD

**Issue by pietro-coretto** [JabRef/jabref#9920]
### JabRef version

5.9 (latest release)

### Operating system

GNU / Linux

### Details on version and operating system

JabRef 5.9 installed via snap on Ubuntu 20.04 LTS

### Checked with the latest development build

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

1. go in `Option > preferences > General`
2. select `Load and save preferences from/to jabref.xml`...
3. then click `save`

After step 3 I am given  the error window 

![image](https://github.com/JabRef/jabref/assets/71322176/cbb07dee-5a29-4d72-b273-5bb8de091b35)

at startup the setting I wanted to save are lost. The file jabref.xml doesn't exist.

### Appendix

...
<details>

<summary>Log File</summary>

```
Paste an excerpt of your log file here
```

</details>

**Comment by Siedlerchr** [comment]
Ah yes this is a known error which should be already fixed in the latest dev version. As a workaround, you have to enter a valid directory for the Main File Directory linked files tab first (even if you don't use it)

To try the latest dev version you can do:
sudo snap install jabref --edge

 It is recommened to make a backup of your library

**Comment by pietro-coretto** [comment]
Yes thanks. I removed the old version of Jabref and installed the latest dev with 
```{sh}
sudo snap install jabref --edge
```
and I can confirm it works now. Issue as resolved, tnx.
